Source/EasyNetQ.Tests/When_using_default_consumer_error_strategy.cs

```diff
@@ -22,7 +22,8 @@ public When_using_default_consumer_error_strategy()
             ErrorQueueNamingConvention = _ => "CustomEasyNetQErrorQueueName",
             ErrorExchangeNamingConvention = info => "CustomErrorExchangePrefixName." + info.RoutingKey,
             ErrorQueueTypeConvention = () => QueueType.Quorum,
-            ErrorExchangeTypeConvention = () => ExchangeType.Topic
+            ErrorExchangeTypeConvention = () => ExchangeType.Topic,
+            ErrorExchangeRoutingKeyConvention = info => info.RoutingKey
```

Comment:
Could you add a custom convention here? Just to test that it works, not the default one.

Reply:
added custom convention for routing key and validated the basicpublish and queuebind call with custom routing key. Please check